tests/regression/77-lin2vareq/18-loop_relational.c

```diff
@@ -0,0 +1,20 @@
+//SKIP PARAM: --set ana.activated[+] lin2vareq
+#include <stdio.h>
+#include <goblint.h>
+
+int main() {
+    int x = 0;
+    int y = 10;
+    int z = 5;
+
+    for(int i = 0; i < 3; i++) {
+        x += z; 
+        y -= i;  
+        z += 2; 
+    }
+
+    __goblint_check(x == 21);  //UNKNOWN!
+    __goblint_check(y == 7);   //UNKNOWN!
+    __goblint_check(z == 11);  //UNKNOWN!
```

Comment:
The program is deterministic. Why are these assertions annotatated with `UNKNOWN!`? How can there be concrete paths where they evaluate to `true` as well as some where they evaluate to `false`?